lib/_http_client.js

```diff
@@ -190,12 +191,18 @@ function ClientRequest(input, options, cb) {
 
   this._ended = false;
   this.res = null;
-  this.aborted = false;
   this.timeoutCb = null;
   this.upgradeOrConnect = false;
   this.parser = null;
   this.maxHeadersCount = null;
 
+  // Used by destroyImpl.
+  this._writableState = {
+    errorEmitted: false,
+    destroyed: false,
+    emitClose: true
+  };
```

Comment:
I don't think this is a good idea. A lot of modules rely on the lack of `_writableState` to detect pseudo-streams.

Reply:
@mcollina: ok, I will find another way.

Reply:
Any suggestion? The idea is to re-use the logic/rules in `destroy()`.

Reply:
How about now?

Reply:
This could work, even though it feels a bit hacky.

Reply:
Best I could think of :).